pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhc-pig";
+  version = "1.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
+    hash = "sha256-MquLChDuJe3DdkxxKV4W0o49IIt7Am+yuhdOqUqexS8=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  nativeBuildInputs = with python3.pkgs; [
```
We can use a with statement so we don't have to add it in front of every dependency.